python/semantic_kernel/connectors/memory/azure_cosmosdb_no_sql/azure_cosmosdb_no_sql_memory_store.py

```diff
@@ -19,6 +21,9 @@
 
 
 @experimental_class
+@deprecated(
```

Comment:
it was decided to not do this quite yet, we'll probably do this in one go right before/during/right after Ignite.